.changeset/warm-bikes-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effect-use/gcp-gcs": minor
```

Comment:
@alex-dixon what do you think we should put here? is this a dev dependency? do we bump? no idea what the pattern is :)